graphql_api/types/commit/commit.graphql

```diff
@@ -26,9 +31,24 @@ type Commit {
   pathContents(path: String, filters: PathContentsFilters): PathContentsResult
   errors(errorType: CommitErrorType!): CommitErrorsConnection!
   totalUploads: Int!
+  # components to be removed with #2286
   components(filters: ComponentsFilters): [Component!]!
   bundleStatus: CommitStatus
   coverageStatus: CommitStatus
+  coverage: CommitCoverage
```

Comment:
What do we think about naming this `coverageAnalytics` so it looks consistent to `repository`?

Reply:
I agree that it makes sense to keep them consistent. Thoughts on something like `coverageFields` and `bundleAnalysisFields` vs `coverageAnalytics` and `bundleAnalysis`?

Reply:
Wait, I just reread this and realized you were talking about consistency with `repository`. Changing it to be coverageAnalytics would make sense.

Reply:
Yeah that's a good question. I think `coverageAnalytics`, `bundleAnalysis`, and `testAnalytics` as the keys whenever we need to split out the products makes sense to me, too!